src/app/components/headerbar/headerbar.ui

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="HeaderBarWidget" parent="AdwBin">
+    <child>
+      <object class="GtkOverlay" id="overlay">
+        <property name="hexpand">1</property>
+        <child>
+          <object class="AdwHeaderBar">
+            <property name="show-end-title-buttons">1</property>
+            <child>
+              <object class="GtkButton" id="go_back">
+                <property name="receives-default">1</property>
+                <property name="halign">start</property>
+                <property name="valign">center</property>
+                <property name="icon-name">go-previous-symbolic</property>
+                <property name="has-frame">0</property>
+              </object>
+            </child>
+            <child type="title">
+              <object class="AdwWindowTitle" id="title">
+                <property name="visible">0</property>
+                <property name="title" translatable="no">Spot</property>
+              </object>
+            </child>
+            <child type="end">
+              <object class="GtkButton" id="start_selection">
+                <property name="receives-default">1</property>
+                <property name="label">Select...</property>
+              </object>
+            </child>
+          </object>
+        </child>
+        <child type="overlay">
+          <object class="AdwHeaderBar" id="selection_header">
+            <property name="show-end-title-buttons">1</property>
```

Comment:
This should be off.

Reply:
we need controls don´t we?!

Reply:
It's expected that within the context of selection mode, window controls aren't shown.

Reply:
Oh alright I missed the context! That seems reasonable